administrator/components/com_contact/src/View/Contacts/HtmlView.php

```diff
@@ -165,9 +166,7 @@ protected function addToolbar()
             $childBar = $dropdown->getChildToolbar();
 
             $childBar->publish('contacts.publish')->listCheck(true);
-
             $childBar->unpublish('contacts.unpublish')->listCheck(true);
-
             $childBar->standardButton('featured')
                 ->text('JFEATURE')
                 ->task('contacts.featured')
```

Comment:
```suggestion
            $childBar->standardButton('featured', 'JFEATURE', 'contacts.featured')
```
Please do the following line as it is not available in the GitHub GUI.

Reply:
@Quy what?

Reply:
both are valid, second makes it shorter

Reply:
It's not just about shorter. the way most IDE's interpret the return types mean that both linters and IDE's interpret stuff better.